H_ecto-models.md

```diff
@@ -609,3 +611,79 @@ end
 ```
 
 At this point, we are completely configured and ready to go. We can go back to the top of this guide and follow along.
+
+#### Using MySQL
```

Comment:
It may be worth noting that you can pass `--database mysql` to `phoenix.new`. Or do we mention it elsewhere?